libr/core/cbin.c

```diff
@@ -1,5 +1,7 @@
 /* radare - LGPL - Copyright 2011-2025 - pancake */
 
+#include "r_io.h"
+#include "r_types.h"
 #define R_LOG_ORIGIN "core.bin"
```

Comment:
```suggestion
#undef R_LOG_ORIGIN
#define R_LOG_ORIGIN "core.bin"
```

and remove the includes from io and types